55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

VarunAgrawalJKT opened the pull request:
Added a new button foe Copy vendor to all estCostMaterial records and fixed the defect to display the correct estimate cost material data when Item type was purchased in the prompt
Files:
Source/est/estCostMaterialList.w
Source/est/EstimateCalcProcs.p

VarunAgrawalJKT committed:
55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

Added a new button foe Copy vendor to all estCostMaterial records and fixed the defect to display the correct estimate cost material data when Item type was purchased in the prompt
Files:
Source/est/estCostMaterialList.w
Source/est/EstimateCalcProcs.p

VarunAgrawalJKT committed:
55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

Change the logic to show all the Error message in one time after processing all the estCostMaterial records.

Files: Source/est/estCostMaterialList.w
       Source/est/EstimateCalcProcs.p

VarunAgrawalJKT committed:
Merge branch 'develop' into feature/55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

VarunAgrawalJKT committed:
55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

Code Review feedback have been implemented
Files:
Source/est/estCostMaterialList.w
Source/est/EstimateCalcProcs.p

VarunAgrawalJKT committed:
55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

Code Review feedback have been implemented
Files:
Source/est/estCostMaterialList.w

VarunAgrawalJKT committed:
Merge branch 'develop' into feature/55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

VarunAgrawalJKT committed:
55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

Fixed the defect to get the correct vendor with Adders.

Files Changed:
est/estCostMaterialList.w
est/EstimateCalcProcs.p
est/EstimateProcs.p

brad-advantzware reviewed (changes requested):
PRIVATE pGetAdders(input..., output array).
	Do iCount = 1 to 6...

PUBLIC Estimate_GetAddersList(input..., output Char).
	def output parameter opcAddersList as character.
	def cAdders as extent...

	Run pGetAdders(input, output cAdders
	do iCount = 1 to EXTENT( opcAddersList	

PUBLIC Estimate_GetAddersArray(Input..., output Array).
	Run pGetAdders

VarunAgrawalJKT committed:
55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

Code review points have been implemented.

Files Changed:
est/estCostMaterialList.w
est/EstimateProcs.p

brad-advantzware committed:
Merge branch 'develop' into feature/55317_CEVersion_New_Enable_Vendor_Selection_when_calculating

# Conflicts:
#	Source/est/EstimateCalcProcs.p
#	Source/est/estCostMaterialList.w
